app/src/screens/SignIn/index.tsx

```diff
@@ -40,20 +39,15 @@ const styles = StyleSheet.create({
 });
 
 const SignIn: React.FC = () => {
-  const { authorize, getCredentials } = useAuth0();
-  const dispatch = useDispatch();
+  const dispatch = useDispatch<Dispatch>();
 
-  const onLogin = async () => {
-    try {
-      await authorize({
-        scope: config.auth0Scope,
-        audience: config.audience,
-      });
-      const { accessToken } = await getCredentials();
-      dispatch(setAccessToken(accessToken));
-    } catch (e) {
-      console.error(e);
-    }
+  React.useEffect(() => {
+    dispatch(checkLoginStatus());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let's leave this out, I think it's fine to include dependencies in the effect hook that shouldn't change. I'm assuming it's `dispatch` that it's concerned might change and I get that it will never change, but I think it's fine to just include it.